safekeeper/src/send_wal.rs

```diff
@@ -566,6 +574,11 @@ impl<IO: AsyncRead + AsyncWrite + Unpin> WalSender<'_, IO> {
                 }))
                 .await?;
 
+            if let Some(appname) = &self.appname {
```

Comment:
Is it added just for testing?
If so I think we should use fail point here...

Reply:
done